gems/policy-parser/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   dev:
-    image: cyberark/ubuntu-ruby-fips:1.0.6
+    image: registry2.itci.conjur.net/cyberark/ubuntu-ruby-fips:20.04-b201fa9
```

Comment:
Should be set to latest after base image release

Reply:
Will rollback before push to master